--- a/arch/arm/boot/dts/kirkwood-iomega_ix2_200.dts	2017-12-18 21:49:39.662253793 +0100
+++ b/arch/arm/boot/dts/kirkwood-iomega_ix2_200.dts	2017-12-18 21:56:13.395817578 +0100
@@ -180,23 +180,22 @@
 
 	partition@0 {
 		label = "u-boot";
-		reg = <0x0000000 0x100000>;
+		reg = <0x0000000 0xa0000>;
 		read-only;
 	};
 
 	partition@a0000 {
-		label = "env";
+		label = "u-boot environment";
 		reg = <0xa0000 0x20000>;
-		read-only;
 	};
 
 	partition@100000 {
-		label = "uImage";
+		label = "kernel";
 		reg = <0x100000 0x300000>;
 	};
 
 	partition@400000 {
-		label = "rootfs";
+		label = "ubi";
 		reg = <0x400000 0x1C00000>;
 	};
 };
